cmd/add_subnet_validator.go

```diff
@@ -119,17 +122,41 @@ func createSubnetValidatorFunc(cmd *cobra.Command, args []string) error {
 		}
 		info.validateStart = time.Now().Add(30 * time.Second)
 		info.validateEnd = end
-		ctx, cancel = context.WithTimeout(context.Background(), requestTimeout)
-		took, err := cli.P().AddSubnetValidator(
-			ctx,
-			info.key,
-			info.subnetID,
-			nodeID,
-			info.validateStart,
-			info.validateEnd,
-			validateWeight,
-		)
-		cancel()
+
+		var took time.Duration
+		if baseWallet != nil {
```

Comment:
Can `InitClient` return a nil base wallet and a nil error? If not, can we remove this check?

Reply:
Is there any reason we can't change the behavior so that failing to create the base wallet will result in returning a non-nil error?